internal/server/v0/placestatvar/golden/get_place_stat_vars/alb.json

```diff
@@ -130,8 +130,6 @@
         "AmountInterestRepayment_Debt_OfficialCreditor_PubliclyGuaranteed_Multilateral_LongTermExternalDebt_LenderWorldBankIDA",
         "AmountInterestRepayment_Debt_PrivateCreditor_PrivateNonGuaranteed_Bond_LongTermExternalDebt",
         "AmountInterestRepayment_Debt_PrivateCreditor_PrivateNonGuaranteed_Bond_LongTermExternalDebt_LenderBondholders",
-        "AmountInterestRepayment_Debt_PrivateCreditor_PrivateNonGuaranteed_OneOrgCommercialBankAndOther_LongTermExternalDebt",
```

Comment:
Is this expected @ajaits ?

Reply:
yes  'OneOrg' was renamed to 'WorldBank' as per OneOrg's request. The SV was thus renamed to:
AmountInterestRepayment_Debt_PrivateCreditor_PrivateNonGuaranteed_WorldBankCommercialBankAndOther_LongTermExternalDebt

Both existed in the last release to allow OneOrg to switch, With infrequent rebuilt, the earlier SV is now deleted.